src/brod_group_coordinator.erl

```diff
@@ -682,19 +682,32 @@ do_commit_offsets_(#state{ groupId                  = GroupId
       },
   Rsp = send_sync(SockPid, Req),
   #kpro_OffsetCommitResponse{oCRspTopic_L = Topics} = Rsp,
-  lists:foreach(
-    fun(#kpro_OCRspTopic{topicName = Topic, oCRspPartition_L = Partitions}) ->
-      lists:foreach(
-        fun(#kpro_OCRspPartition{partition = Partition, errorCode = EC}) ->
-          kpro_ErrorCode:is_error(EC) andalso
-            begin
+  ErrorSet = lists:foldl(
```

Comment:
oh, please use the ESCALATE_EC macro instead of erlang:throw.
-- to make code search easier.